packages/rolldown/tests/fixtures/input/array/_config.ts

```diff
@@ -4,7 +4,8 @@ import { expect } from 'vitest'
 import { getOutputChunkNames } from '@tests/utils'
 
 const config: RollupOptions = {
-  input: [path.join(__dirname, 'main.js'), path.join(__dirname, 'entry.js')],
+  input: ['main.js', 'entry.js'],
+  cwd: __dirname,
```

Comment:
Since you already pass `cwd` in `packages/rolldown/tests/fixture.test.ts`, this is unnessary to pass it again.

Reply:
Done!